packages/notifi-frontend-client/lib/models/FusionEvent.ts

```diff
@@ -0,0 +1,61 @@
+import { Types } from '@notifi-network/notifi-graphql';
+
+export type FusionEventMetadata = {
+  uiConfigOverride?: {
+    topicDisplayName?: string;
+    historyDisplayName?: string;
+    icon?: Types.GenericEventIconHint;
+    customIconUrl?: string;
+  };
+  filters: Array<AlertFilter>;
+};
+
+/**
+ * @param name - `string` unique name
+ */
+export type AlertFilter = AlertFilterBase | AlertFrequencyFilter;
+export type AlertFilterBase = {
+  name: string;
+  type: AlertFilterType;
+  executionPriority: number;
+  userInputParams: UserInputParam<UiType>[];
+  staticFilterParams?: Record<string, object | string | number>;
+  requiredParserVariables: Array<RequiredParserVariable>;
+};
+
+export type AlertFrequencyFilter = AlertFilterBase & {
```

Comment:
I think type of AlertFequencyFilter should be 
{
      name: string,
      minimumDurationBetweenTriggersInMinutes: number,
      executionPriority: number,
}